index.html

```diff
@@ -2179,7 +2256,9 @@ <h1>Directory Service API Specification</h1>
             The <a>TM</a> should describe all interaction affordances and indicate which ones
             are required (i.e. mandatory by this spec). Moreover, it should use placeholders
             for deployment-specific attributes.
+            Informative affordances (JSONPath and XPath) should be defined as such.
         </p>
+
         The API of the directory is specified as a <a>Thing Model</a>.
         The Thing Model alone should not be considered as the reference
         to implement or interact with a directory.
```

Comment:
A little unclear.  Should emphasize that all Thing Model constraints are normative, but there are *additional* constraints and assertions given in the body of the specification that also need to be followed by implementations.